### ISSUE TITLE
Occupancy issues in WTP case study

### ISSUE THREAD

**Issue by martin-mosteiro** [architecture-building-systems/CityEnergyAnalyst#1408]
During the test run, the following issues came up with the `occupancy.dbf` files for every scenarioa of the WTP case study:
1-All buildings were assigned occupancy types (namely, 50% OFFICE and 50% MULTI_RES) regardless of the case study's focus (i.e., CBD, RES or MIX). This might be on purpose if setting up the case study is part of the exercises in the Singapore case study. Maybe @shizhongming can clarify?
2-All case studies had the following possible occupancy types: OFFICE (set to 0.5 by default), MULTI_RES (set to 0.5 by default) and RETAIL (set to 0 by default). However, it seemed RETAIL was assigned a format with no decimals, so that if the value was set to, say, 0.2 on ArcScene, it would automatically revert back to 0 when saved (even after changing the number of decimal places allowed on ArcScene). Perhaps this could be solved by assigning values of 0.0 to RETAIL instead of 0?

**Comment by JIMENOFONSECA** [comment]
@shizhongming could you change the case studies and update?

**Comment by shizhongming** [comment]
@martin-mosteiro I think your case study folder is not up-to-date....I figured the problem out two weeks before. It is caused by a bug of the Create_a_new_project script...Please sync and it should help~ :)

**Comment by martin-mosteiro** [comment]
Great!

**Comment by martin-mosteiro** [comment]
@shizhongming The updated case studies look good! Only issue is that columns that have 0 of an occupancy type get assigned `int` as a data type. Which means from then on the building can only have 0% of that function or 100%. Practical example: 02_RES_medium_density has 0% office in all the buildings. That column can now only be assigned values of 0 or 1 on ArcScene.

Namely, here are the problematic scenarios, the problematic columns, and the problematic dtype:
```
02_RES_medium_density OFFICE int64
03_MIX_medium_density RETAIL int64
MIX_high_density RETAIL int64
MIX_low_density RETAIL int64
RES_high_density OFFICE int64
RES_low_density OFFICE int64
```
I just fixed this for you and I'll be doing a pull request in the next 1 minute. But keep this in mind for the future!

**Comment by martin-mosteiro** [comment]
This issue is solved by PR #34 on the cea-reference-case repository:
https://github.com/architecture-building-systems/cea-reference-case/pull/34
